### PR TITLE
refactor: mjml before release

### PR DESCRIPTION
**Pull request by bashleigh**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #

### CODE REVIEW

File: packages/cognito-custom-mail-lambda/package.json [hunk 1]
```diff
@@ -7,8 +7,9 @@
     "dist"
   ],
   "scripts": {
+    "generate-html": "yarn ts-node src/scripts/generate.ts",
```
Comment (willmcvay) [inline packages/cognito-custom-mail-lambda/package.json:10]: Why not just put this in the build step then you don't need to commit the html and you know you always have the latest compiled templates before releasing?

File: packages/cognito-custom-mail-lambda/src/mailer/templates/confirm-registration.html [hunk 2]
```diff
@@ -0,0 +1,368 @@
+<!doctype html>
```
Comment (willmcvay) [inline packages/cognito-custom-mail-lambda/src/mailer/templates/confirm-registration.html:1]: See my point above, you can then remove the html files

**Review by willmcvay — CHANGES_REQUESTED** [review]

File: packages/cognito-custom-mail-lambda/package.json (hunk 1 above)
Comment (bashleigh) [reply packages/cognito-custom-mail-lambda/package.json:10]: yea guess so, needed them for tests 😭

**Review by willmcvay — APPROVED** [review]
LGTM

**Pull request merged**